backend/contracts/Quiz.sol

```diff
@@ -36,18 +47,19 @@ contract Quiz {
     address private _owner;
     // Encryption key for encrypting payout certificates.
     bytes32 private _key;
-    // Quiz ID for generating NFT images
-    uint32 private _quizID;
     // Base64 svg image for this quiz instance
     string private _svgImage;
     // NFT contract address
     address private _nftAddress;
```

Comment:
Let's make this public, so we can get rid of VITE_NFT_ADDRESS env variable for deploying the frontend.

Reply:
Changed nftAddress to public.